mcpjam-inspector/.env.production

```
@@ -3,6 +3,6 @@ CONVEX_URL=https://outstanding-fennec-304.convex.cloud
 VITE_WORKOS_CLIENT_ID=client_01K4C1TVPBE7JTBFQJF9SDW9P9
 VITE_WORKOS_REDIRECT_URI=mcpjam://oauth/callback
 CONVEX_HTTP_URL=https://outstanding-fennec-304.convex.site
-ENVIRONMENT=production
+ENVIRONMENT=prod
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENVIRONMENT key should go before the VITE_CONVEX_URL key

(UnorderedKey)